--- superfish.html.twig.mzu	2018-04-05 17:01:07.808219475 +0200
+++ superfish.html.twig	2018-04-05 17:00:02.636573094 +0200
@@ -0,0 +1,19 @@
+{#
+/**
+ * @file
+ * Default theme implementation of the Superfish menus.
+ *
+ * Available variables:
+ * - id: Unique menu identifier.
+ * - menu_classes: Menu classes.
+ * - menu_items: Menu tree.
+ *
+ * @see template_preprocess_superfish()
+ *
+ * @ingroup themeable
+ */
+#}
+
+<ul id="{{ id }}" class="{{ menu_classes }}">
+  {{ menu_items }}
+</ul>
